config/nginx.conf

```diff
@@ -16,9 +16,6 @@ http {
     # Log to stderr, so we can see stacktraces in PaaSTA logs
     error_log /dev/stderr error;
 
-    log_format main_spectre '{"time_iso8601": "$time_iso8601", "remote_addr": "$remote_addr", "connection": "$connection", "connection_requests": "$connection_requests", "http_user_agent": "$http_user_agent", "status": "$status", "request_length": "$request_length", "bytes_sent": "$bytes_sent", "request_time": "$request_time", "request_uri": "$request_uri", "unix_time": "$msec", "x_smartstack_source": "$http_x_smartstack_source", "x_smartstack_destination": "$http_x_smartstack_destination", "spectre_cache_status": "$sent_http_spectre_cache_status"}';
-    access_log /dev/stdout main_spectre;
```

Comment:
might have to put access_log /dev/null here instead because the default behaviour of nginx in foreground would be to spit out the logs to stdout. Ideally we could even make this some kind of parameter so we don't have to change the code if we want to toggle it back on

Reply:
Not super elegant though
https://serverfault.com/questions/577370/how-can-i-use-environment-variables-in-nginx-conf